.config/language.config.php

```diff
@@ -0,0 +1,7 @@
+<?php
+$CONFIG = array(
+  'default_language' => getenv('DEFAULT_LANGUAGE') ?: 'en',
```

Comment:
I'm not sure that `'en'`  is the right default here. https://github.com/nextcloud/server/search?q=getSystemValue%28%27default_language%27%2C&unscoped_q=getSystemValue%28%27default_language%27%2C It seems `default_language` is request with `false` or `en` as default. I think we should try to keep the way of returning a default value for configuration options not defined. If I understand https://github.com/nextcloud/server/blob/5bf3d1bb384da56adbf205752be8f840aac3b0c5/lib/private/Config.php#L95-L99 correctly a existing value in $this->cache will used if isset evaluates to true. As per https://www.php.net/manual/en/types.comparisons.php `isset(null)` will be false hence the right default value here. Probably needs a second opinion because the "configuration management" has it pitfalls :see_no_evil:

Reply:
What is your recommendation then? I don't have a strong opinion.
(But I agree that in an ideal world, we don't create logic that is not present on the underlying logic - but anyway, as you see the underlying logic is not crystal clear either so 🤷‍♂ )
(And thanks for taking the time to review such details ;) )

Reply:
`null` because `isset(null)` is false hence the default (in the code) should be used. That's nearly the same as option not set (at least I assume that)

Reply:
Should we include this at all? Custom config can always be added, but I don't want to copy (and maintain) Nextcloud defaults for every option.

Reply:
No. Sorry I was wrong. If we fallback to null it's not possible to overwrite it via config.php anymore. We run into this recently with my disable mod remoteip change. 

I think we should set if defined via env and do nothing otherwise.